joern-cli/frontends/gosrc2cpg/src/test/scala/io/joern/go2cpg/dataflow/ArrayDataflowTests.scala

```diff
@@ -0,0 +1,24 @@
+package io.joern.go2cpg.dataflow
+import io.shiftleft.semanticcpg.language._
+import io.joern.dataflowengineoss.language._
+import io.joern.go2cpg.testfixtures.GoCodeToCpgSuite
+
+class ArrayDataflowTests extends GoCodeToCpgSuite {
+  "Source to sink dataflow through arrays" should {
+    // TODO: Investigate why this is not working.
+    "be reachable (case 1)" ignore {
+      val cpg = code("""
+          |package main
+          |func main() {
+          | a=1
```

Comment:
I believe this is not a valid goLang code. As the variable `a` is not defined and directly initialised.

Reply:
My bad. The test still does not pass though.